app/selenium_ui/jira/modules.py

```diff
@@ -130,7 +128,7 @@ def __filer_epic(element):
             webdriver.find_element_by_id(ISSUE_TYPE_DROPDOWN).click()
             issue_elements_in_dropdown = webdriver.find_elements_by_class_name("aui-list-item")
             if issue_elements_in_dropdown:
-                filtered_issue_elements = list(filter(__filer_epic, issue_elements))
+                filtered_issue_elements = list(filter(__filer_epic, issue_elements_in_dropdown))
```

Comment:
I believe this was fixed in another PR.
Just merge with dev to get this fix, but not make it here.

Reply:
I found out that I've not changed this variable.